Ubuntu-Installing-DevTools-and-Modern-Web-Browsers.md

```diff
@@ -0,0 +1,45 @@
+# Installing DevTools and modern web browsers
```

Comment:
Just `Ubuntu-Installing-DevTools.md` should do. The version of firefox and chromium are devtools and modem browsers at the same time.